commons/commons-util/src/main/kotlin/ebi/ac/uk/io/ext/FileExtensions.kt

```diff
@@ -0,0 +1,17 @@
+package ebi.ac.uk.io.ext
+
+import ebi.ac.uk.io.FileUtils
+import java.io.File
+import java.nio.file.Files
+
+fun File.notExist() = Files.exists(toPath()).not()
+
+fun File.asFileList(): List<File> = FileUtils.listFiles(this)
+
+fun File.size() = FileUtils.size(this)
+
+fun File.createDirectory(name: String): File = Files.createDirectory(toPath().resolve(name)).toFile()
+
+fun File.createNewFile(name: String): File = resolve(name).apply { createNewFile() }
+
+fun File.createNewFile(name: String, text: String): File = resolve(name).apply { createNewFile(); writeText(text) }
```

Comment:
Why don't leave each method call on its own line? A semi-colon looks very odd in Kotlin :sweat_smile: